tests/numeric_test.py

```diff
@@ -814,30 +955,48 @@ def testArctan2(self):
         pda2 = ak.array(na2)
 
         self.assertTrue(
-            np.allclose(np.arctan2(na1, na2), ak.arctan2(pda1, pda2).to_ndarray(), equal_nan=True)
```

Comment:
Can you check your line length setting in black?  It seems like more lines are being reformatted than necessary.